src/ziggurat/messaging/consumer.clj

```diff
@@ -21,7 +21,7 @@
     (catch Exception e
       (log/info "old message format read, converting to message-payload: " message)
       (let [retry-count (:retry-count message)
-            message-payload (mpr/->MessagePayload (dissoc message :retry-count) topic-entity)]
+            message-payload (mpr/->MessagePayload (dissoc message :retry-count) topic-entity {})]
```

Comment:
why are we passing empty {} here? why not have headers as an optional parameter?

Reply:
ok have added that as optional param to the schema